### ISSUE TITLE
Trying to restore iOS 10.3.3 on iPad Air (WiFi) always getting: Device did not reconnect. Possibly invalid iBEC.

### ISSUE THREAD

**Issue by tobia** [marijuanARM/futurerestore#72]
Hello

I'm trying to downgrade an iPad Air 1, WiFi-only model, to iOS 10.3.3 using https://github.com/LukeZGD/iOS-OTA-Downgrader, but no matter what I do, it always fails to reconnect after iBEC is sent.

I tried several versions and forks of futurerestore, to no avail.

Here is a --debug log using the latest test build:

[fr-log.txt](https://github.com/futurerestore/futurerestore/files/8403280/fr-log.txt)

Does anybody have any idea how to solve this issue?

I noticed that as soon as fr quits, Finder pops up asking to restore the iPad. Is this expected? Maybe the timeout for "waiting for device to reconnect" is too strict?

**Comment by Cryptiiiic** [comment]
Do you have m1 Mac?

**Comment by Cryptiiiic** [comment]
Regardless afaik a7 needs ––no–ibss flag

**Comment by tobia** [comment]
No, I tried with two different x86 Macs, one Core i5 with Catalina and a Macbook Air with Big Sur.

I tried adding --no-ibss but every time I do, the upload gets stuck at 78%:

```
Device found in DFU Mode.
Sending iBEC (668704 bytes)...
[========================================          ]  78.4%Cleaning up...
[exception]:
what=ERROR: Unable to send iBEC component: Unable to upload data to device
```

**Comment by verygenericname** [comment]
send a pwnediBSS then do noibss and see how it goes

**Comment by Cryptiiiic** [comment]
You need to make sure you ran fugu rmsigchks  or ipwnder-lite for pwned iBSS to work in the first place

**Comment by tobia** [comment]
I'm not sure what was wrong in the iPad, but something was wrong.

I tried to restore it using Finder, many times, but was always getting the infamous error "9". I finally succeeded using Apple's restore procedure, but only by leaving a pair of headphones plugged in during the entire procedure, a weird recommendation I found on an obscure article. I don't know if it's just a coincidence, of it's some sort of backdoor that Apple intentionally left there to circumvent some of their checks.

In any case, after completing the official Apple restore, I was able to flash my image of choice using futurerestore. All is well.
